charts/amazon-cloudwatch-observability/templates/target-allocator-clusterrolebinding.yaml

```
@@ -0,0 +1,21 @@
+{{- if .Values.agent.enabled }}
+{{- range $i, $customAgent := .Values.agentWorkloads }}
+{{- if and (eq $customAgent.mode "statefulset") (or $customAgent.prometheus.config $customAgent.prometheus.customResourceName) }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  labels:
+    {{- include "amazon-cloudwatch-observability.labels" $ | nindent 4 }}
+  name: "cloudwatch-agent-target-allocator-rolebinding"
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: "cloudwatch-agent-target-allocator"
```

```suggestion
  name: "cloudwatch-agent-target-allocator-role"
```
The role is named `cloudwatch-agent-target-allocator-role` it can be seen [here](https://github.com/aws-observability/helm-charts/blob/b83c1dfbac899a5dc5e07e7ce195dc0e2d3c274f/charts/amazon-cloudwatch-observability/templates/target-allocator-clusterrole.yaml#L9)